### PR TITLE
Feature/stij 237: Edit forms grid

### PR DESCRIPTION
**Pull request by meirege**
## Description
Changes to the form components. Remove all bootstrap related styling for forms.

## Motivation and Context
It's simply not maintainable in a Drupal environment to use bootstraps in forms.

## How Has This Been Tested?
Through browser testing locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: components/21-atoms/fieldset/_fieldset.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fieldset .fieldset-wrapper {
```
Comment (delrueba) [inline components/21-atoms/fieldset/_fieldset.scss]: Why is the wrapper inside of the element?

File: components/31-molecules/fields/_fields.scss [hunk 2]
```diff
@@ -127,36 +127,42 @@ input[type='checkbox'] {
...
+        margin-right: $gutter-width;
       }
 
-      &:nth-child(1) {
+      width: 100%;
+
+      &:first-child {
         @include tablet {
-          float: right;
+          order: 2;
```
Comment (delrueba) [inline components/31-molecules/fields/_fields.scss]: I suggest using @include tablet only once for first- & last-child

File: components/41-organisms/inline-form-groups/address/_address.scss [hunk 3]
```diff
@@ -1,31 +1,59 @@
...
+      &.address-box-number,
+      &.address-city,
+      &.address-nim {
+        margin-right: 0;
+      }
     }
   }
+
+  .form-columns .form-item-column {
+    width: 100% !important; // sass-lint:disable-line no-important
```
Comment (delrueba) [inline components/41-organisms/inline-form-groups/address/_address.scss]: Am I correct to assume these classes are meant to disappear in version 3.0.0?
Best add a ticket / todo then.

File: components/41-organisms/inline-form-groups/address/address--address-with-errors.twig [hunk 4]
```diff
@@ -3,148 +3,148 @@
...
+        "field_error": "Please fill in a valid city",
+        "description": "",
+
+        "textfield_id": "address-city",
+        "textfield_name": "address-city",
+        "modifier": "error"
+      }%}
+
+      {% include '@select'
         with {
```
Comment (delrueba) [inline components/41-organisms/inline-form-groups/address/address--address-with-errors.twig]: Indentation (same for the blocks above)

File: components/61-layouts/forms/form-with-steps/_form-with-steps.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.multistepform {
+  .form-sidebar  {
+    width: span(3);
+    margin-right: gutter();
+  }
+
+
+
```
Comment (delrueba) [inline components/61-layouts/forms/form-with-steps/_form-with-steps.scss]: whitelines

File: components/21-atoms/fieldset/_fieldset.scss (hunk 1 above)
Comment (meirege) [reply components/21-atoms/fieldset/_fieldset.scss]: It's a drupal related thing. A fieldset has a legend and under it a wrapper with it's actual content in it.

File: components/31-molecules/fields/_fields.scss (hunk 2 above)
Comment (meirege) [reply components/31-molecules/fields/_fields.scss]: That goes against the way we use it everywhere else. :)
Every selector has it's own mixins for breakpoints. It keeps it more readable.

File: components/41-organisms/inline-form-groups/address/_address.scss (hunk 3 above)
Comment (meirege) [reply components/41-organisms/inline-form-groups/address/_address.scss]: Not that I know of.

File: components/31-molecules/fields/_fields.scss (hunk 2 above)
Comment (delrueba) [reply components/31-molecules/fields/_fields.scss]: OK, I'm all for comformity

File: components/41-organisms/inline-form-groups/address/address--address-with-errors.twig (hunk 4 above)
Comment (meirege) [reply components/41-organisms/inline-form-groups/address/address--address-with-errors.twig]: Fixed.

File: components/61-layouts/forms/form-with-steps/_form-with-steps.scss (hunk 5 above)
Comment (meirege) [reply components/61-layouts/forms/form-with-steps/_form-with-steps.scss]: Fixed.

File: components/41-organisms/inline-form-groups/address/_address.scss (hunk 3 above)
Comment (delrueba) [reply components/41-organisms/inline-form-groups/address/_address.scss]: If you don't want columns, don't add columns.

**Pull request merged**